tilt-provider.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "aws",
+    "config": {
+        "image": "gcr.io/k8s-staging-cluster-api-aws/cluster-api-aws-controller",
+        "ignore": [
+            "config/manager/credentials.env"
```

Comment:
Should this be `config/manager/credentials` instead?

Reply:
It's actually not needed anymore, because I'm using `only` in the Tiltfile. Will remove.